extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java

```diff
@@ -0,0 +1,68 @@
+package io.quarkus.swaggerui.deployment;
+
+import javax.inject.Inject;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.arc.deployment.BeanContainerBuildItem;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.LaunchModeBuildItem;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.quarkus.swaggerui.runtime.SwaggerUiTemplate;
+import io.quarkus.undertow.deployment.ServletExtensionBuildItem;
+
+public class SwaggerUiProcessor {
+
+    private static final Logger log = Logger.getLogger(SwaggerUiProcessor.class.getName());
+
+    /**
+     * The configuration for swagger-ui.
+     */
+    private SwaggerUiConfig swaggerUiConfig;
+
+    @Inject
+    private LaunchModeBuildItem launch;
+
+    /**
+     * Register this extension as a swagger-ui feature
+     *
+     * @return
+     */
+    @BuildStep
+    FeatureBuildItem feature() {
+        log.info("Feature: swagger-ui");
+        return new FeatureBuildItem("swagger-ui");
```

Comment:
This should be added as a constant in `FeatureBuildItem`. At least, that's the convention for now. Please follow the alphabetical order there.

Reply:
Log removed + Constant introduced